dev-java/rxtx/rxtx-2.2_pre2-r2.ebuild

```diff
@@ -34,11 +41,7 @@ src_prepare() {
 		-e "s:-source ... -target ...:$(java-pkg_javac-args):g" \
 		configure.in || die
 
-	epatch \
-		"${FILESDIR}/${PN}-2.1-7r2-lfd.diff" \
-		"${FILESDIR}/${PN}-2.1-7r2-nouts.diff" \
-		"${FILESDIR}/${P}-add-ttyACM.patch" \
-		"${FILESDIR}/${P}-limits.patch"
+	default
 
 	rm acinclude.m4 || die
 	eautoreconf
```

Comment:
this one fails for me with java 11:
```
/bin/sh: line 1: /usr/lib64/openjdk-11/bin/javah: No such file or directory
```
the path should be fixed.

Reply:
The patch below fixes the path issue, but then I get new errors like `javah is not available for openjdk-11-bin on x86_64`.

Any thoughts on how to resolve this?

```diff
--- a/dev-java/rxtx/rxtx-2.2_pre2-r2.ebuild
+++ b/dev-java/rxtx/rxtx-2.2_pre2-r2.ebuild
@@ -39,6 +39,7 @@ src_prepare() {
 		-e "s:UTS_RELEASE::g" \
 		-e "s:AM_CONFIG_HEADER:AC_CONFIG_HEADERS:g" \
 		-e "s:-source ... -target ...:$(java-pkg_javac-args):g" \
+		-e 's|JAVAH="\\$(JPATH)|JAVAH="'"${EPREFIX}/usr|" \
 		configure.in || die
 
 	default

```

Reply:
@fordfrog since the scope of the PR is to update the `java-utils-2.eclass`, and I'm dealing with jvm-11 things as time permits: should we still merge this commit and deal with the jvm-11 elsewhere?

Reply:
i will have a look at it.

Reply:
so i searched for that and [here](https://stackoverflow.com/questions/54020303/open-jdk-11-and-javah-in-pom-xml)'s the explanation. the easiest fix is to restrict `DEPEND` to `virtual/jdk:1.8` and keep `RDEPEND` as is. that is, for building java 1.8 will be used as it has working `javah`, but it can run even on newer java.

Reply:
Got it - I've made the change to `DEPEND`, however it still uses the system-wide (via `eselect java-vm set...`) jvm to try to build. which happens to be jvm-11.

Is there a way to specify in the ebuild which jvm to build with?

It seems that `JAVA_PKG_FORCE_VM` would accomplish this, but the documentation specifically says not to use it in the final ebuild.